Oracle/scripts/split_files.sh

```diff
@@ -0,0 +1,98 @@
+## ----- Function to move files
+move_file() {
```

Comment:
Since this method does not take any arguments and is always moving files to the output path, what about renaming it to something like: 

```suggestion
move_file_to_output() {
```

Reply:
Done